packages/utils/src/crypto.js

```diff
@@ -0,0 +1,7 @@
+import crypto from 'crypto'
+
+export const createMd5HashFromString = (string, length = 20) => {
+  return crypto.createHash('md5').update(string).digest('hex').substr(0, length)
+}
+
+export const createTimeHash = () => createMd5HashFromString(new Date().toISOString())
```

Comment:
Why hashing date in an ireversable way? By using a timestamp we can compare/sort builds by date.

Reply:
Right, that would be possible. But that would also mean that every Nuxt application will have the timestamp of the last build/generate automatically inserted. Not sure if that's a good idea

Reply:
Hmm what's the concern of not exposing timestamp of last build? Hash is purely based on time. Except that we obfuscate it for an unknown reason. But can sometimes be useful.

Reply:
I'm a huge fan privacy by design and I'd consider the last build time of a website as a partially sensible data point. You could infer a lot from that, not only because most of the builds happen automatically after a commit.

But we could give the user the option override the "hash" (or build "token") creation method so every user can decide on their own

Reply:
File creation time can be also exposed by the other ways (like last modified header of static generated HTML files)

While seeing the point, I don't think that is really a (at least security) concern.

Reply:
Agree with manni, privacy by design means only disclosing information when you really need to. What might not be interesting today could be that one essential bit of information needed tomorrow. File created/modified times could already be masked by eg a recursive touch, so its also all about having a choice.

Reply:
In this case, privacy/security is overseen. By default, build dates are already exposed. If that is a privacy concern could be discussed separately. 

![image](https://user-images.githubusercontent.com/5158436/62011125-a3e76e80-b189-11e9-968d-17dc1f923e8e.png)

I'm OK with adding option but not hashing by _default_ makes builds easier to understand and maintain without exposing (at least a new) concern.